presto-main/src/test/java/com/facebook/presto/block/TestSliceArrayBlock.java

```diff
@@ -30,6 +34,30 @@ public void test()
     }
 
     @Test
+    public void testReferenceCounting()
+    {
+        int positionCount = 5;
+        Slice[] baseValues = createExpectedUniqueValues(positionCount);
+        Slice[] identicalValues = createExpectedUniqueValues(positionCount);
+        Slice[] realValues = new Slice[positionCount * 3];
+        long sliceContentBytes = 0;
+        // create a reference and an identical value copy
```

Comment:
This comment doesn't tell much. If you really want to explain what's going on maybe we can say something along the lines of `the first and second five slices point to baseValues, and the third five slices point to identicalValues`.